app/_data/plugins/otel-metrics.yaml

```diff
@@ -0,0 +1,365 @@
+metrics:
+  - name: http.server.request.count
+    description: Total number of incoming HTTP requests.
+    unit: "{request}"
+    type: Sum
+    attributes:
+      - kong.service.name
```

Comment:
Lack of `kong.response.status_code` attribute, the value of it is "response HTTP status code."